api/subscribe.py

```diff
@@ -110,18 +118,24 @@ def poster(each_group_id: str, each_author_username: str, name: str) -> None:
             context.bot.send_message(
                 chat_id=each_group_id,
                 text=f"@{each_author_username} error occurred while fetching posts from /r/{name}. Automatically "
-                     f"removing subscription.",
+                f"removing subscription.",
                 parse_mode="html",
             )
 
     for group_id, subreddit_name, author_username in cursor.fetchall():
-        context.dispatcher.run_async(poster, group_id, author_username, subreddit_name)
+        try:
+            context.dispatcher.run_async(
+                poster, group_id, author_username, subreddit_name
+            )
+        except Unauthorized:
```

Comment:
Raising `Unauthorized` at L105 would automatically take care of filtering out broken deliveries.